src/vs/editor/common/languageSelector.ts

```diff
@@ -120,6 +125,10 @@ export function score(selector: LanguageSelector | undefined, candidateUri: URI,
 			}
 		}
 
+		if ((selector as LanguageFilter).isBuiltin) {
+			ret = Math.max(ret - 0.5, 0);
+		}
+
```

Comment:
This looks fine but changing the score value might have unforeseen consequences. We do export this function as API. Instead (and similar to time) we could take the builtin property only into account when comparing (and needing a tiebreaker), e.g here: [_compareByScoreAndTime](https://github.com/microsoft/vscode/blob/949c8073e29554a897d764dcd05930bcb626e377/src/vs/editor/common/languageFeatureRegistry.ts#L184)

Reply:
Thanks. I've moved the logic to that file instead